examples/demo2/tabs/weather.rs

```diff
@@ -1,8 +1,14 @@
 use itertools::Itertools;
 use palette::Okhsv;
 use ratatui::{
-    prelude::*,
-    widgets::{calendar::CalendarEventStore, *},
+    buffer::Buffer,
+    layout::{Constraint, Direction, Layout, Margin, Rect},
+    style::{Color, Style, Stylize},
+    symbols::{self},
```

Comment:
https://rust-lang.github.io/rust-clippy/master/index.html#/unnecessary_self_imports
```suggestion
    symbols,
```

Reply:
fixed

Reply:
How about adding that lint? This one is fixed now, the lint will ensure it being fixed in the future too.

The described edge case if this lint sounds like fishy code to me that should be renamed so when this lint results in a naming conflict, that should be fixed rather than keeping `::{self}`

Reply:
done